lessons/module-2/github-collaboration.md

```diff
@@ -7,21 +7,22 @@ module: 2
 
 ## Goals
 
-By the end of this lesson, students should be able to:
-
-* Give professional and actionable feedback through code reviews
-* Recognize what makes a README useful
-* File and organize issues with labels, that are closed through PRs
+__      __.__            __     ____ _____________   ____ _______________     ___________________________________________________
```

Comment:
This is fun, but I don't think it should replace the goals of the lesson, as students find those useful.